packages/cozy-clisk/src/launcher/saveFiles.js

```diff
@@ -154,29 +164,17 @@ const saveEntry = async function (client, entry, options) {
   return entry
 }
 
-function noMetadataDeduplicationWarning(options) {
+function noMetadataDeduplicationErrors(options) {
   const fileIdAttributes = options.fileIdAttributes
   if (!fileIdAttributes) {
-    log.warn(
-      'No deduplication key is defined, file deduplication will be based on file path'
+    throw new Error(
+      'Savefiles : No deduplication key is defined, file deduplication will be based on file path'
     )
   }
 
   const slug = get(options, 'manifest.slug')
   if (!slug) {
-    log.warn(
-      'No slug is defined for the current connector, file deduplication will be based on file path'
-    )
-  }
-
-  const sourceAccountIdentifier = get(
-    options,
-    'sourceAccountOptions.sourceAccountIdentifier'
-  )
-  if (!sourceAccountIdentifier) {
-    log.warn(
-      'No sourceAccountIdentifier is defined in options, file deduplication will be based on file path'
-    )
+    throw new Error('Savefiles : No slug is defined for the current connector')
   }
 }
 
```

Comment:
should be tagged as a BREAKING CHANGE even if you think that all clisk will not be impacted by this change.

Reply:
Done